h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostUtils.java

```diff
@@ -165,32 +165,22 @@ private static int setResponseAndWeight(Vec.Reader w, float[] resp, float[] weig
     /**
      * convert a set of H2O chunks (representing a part of a vector) to a sparse DMatrix
      * @param response name of the response column
-     * @param weight name of the weight column
-     * @param fold name of the fold assignment column
      * @return DMatrix
      * @throws XGBoostError
      */
     public static DMatrix convertChunksToDMatrix(DataInfo di,
```

Comment:
OMG, does this mean we were not generating predictions for 0 weighted rows before?

Reply:
since the argument was never used so no

Reply:
ok, phew